pkg/skaffold/docker/client.go

```diff
@@ -34,23 +34,23 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-type ImageAPIClient interface {
-	client.ImageAPIClient
+type DockerAPIClient interface {
+	client.CommonAPIClient
```

Comment:
We could lock this down to ImageAPIClient + RegistryAPIClient, but I think this is fine.